source/javascripts/controllers/_YMLController.js.erb

```diff
@@ -12,7 +12,7 @@ import { configureMonacoYaml } from 'monaco-yaml';
 			var model;
 
 			const defaultSchema = {
-				uri: 'https://raw.githubusercontent.com/bitrise-io/bitrise-json-schemas/main/bitrise.schema.json',
+				url: 'https://raw.githubusercontent.com/SchemaStore/schemastore/master/src/schemas/json/bitrise.json',
```

Comment:
Why don't we use the official schema store URL (`https://json.schemastore.org/bitrise.json`) ?